AuthenticatorPro/Source/Data/Authenticator.cs

```diff
@@ -23,7 +24,7 @@ public class Authenticator
         public const int DefaultDigits = 6;
         public const int DefaultPeriod = 30;
 
-        public const int MinDigits = 6;
+        public const int MinDigits = 5;
```

Comment:
This change allows 5 digit codes for TOTP and HOTP which I'm not sure is really a thing. But don't worry about it, I may implement specific digit rules for different AuthenticatorTypes.

Reply:
I was aware of this, but am also hesitant to add `type != AuthenticatorType.SteamOtp` checks to the validity checks. I thought about digits rules per authenticator type as well but it's a larger change than this scope.

Reply:
Thinking again, this can be reverted if we enforce that a Steam authenticator will always be five digits, since this change is only to circumvent the check in `Authenticator`

Reply:
I'm thinking it might be best to move this outside of the Authenticator class. Instead AuthenticatorType should be responsible for the min and max digits. In the case of Steam, min and max would both be 5. But since this change doesn't break anything it's not a huge concern at the moment :)